src/main/java/epam/task/gymbootdb/handler/GymExceptionHandler.java

```diff
@@ -54,16 +69,62 @@ public ResponseEntity<Map<String, Object>> handleValidationException(MethodArgum
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(body);
     }
 
+    @ExceptionHandler(AuthenticationException.class)
```

Comment:
Maybe it's better to extract all auth and jwt handlers to the separate controller advice